test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -587,7 +587,7 @@ public void RankingLightGBMTest()
             Done();
         }
 
-        [Fact]
+        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))] // x86 fails with "Unknown command: 'train'; Format error at (83,3)-(83,4011): Illegal quoting"
```

Comment:
This should be investigated and fixed.

Reply:
Was there a follow up issue for this?

---
In reply to: [222758690](https://github.com/dotnet/machinelearning/pull/1008#discussion_r222758690) [](ancestors = 222758690)

Reply:
I've not logged issues for anything that requires followup yet.

Reply:
https://github.com/dotnet/machinelearning/issues/1216